moveit_ros/planning/CHANGELOG.rst

```diff
@@ -2,9 +2,6 @@
 Changelog for package moveit_ros_planning
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-2.5.1 (2022-05-31)
-------------------
-
```

Comment:
I don't think you want to change this. This is probably just a rebase mistake?

Reply:
Yeah, that was not meant to be changed.